### PR TITLE
[FEAT] mongo repl and standalone configs in colibri server with symbo…

### PR DESCRIPTION
**Pull request by martynas-openg**
Not yet tested on production;
Will propably remove backuping the mongod.conf file task after testing on production because it becomes a symlink;

### CODE REVIEW

File: templates/mongod.standalone.conf.j2 [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  path: /var/log/mongodb/mongod.log
+  logAppend: true
+  destination: file
+
+processManagement:
+  pidFilePath: /var/run/mongodb/mongod.pid
+  fork: true
+
+net:
+  bindIp: {% if mongodb_internal_vpn_ip is defined %}{{ mongodb_internal_vpn_ip }},{% endif %}127.0.0.1
```
Comment (Ben10k) [inline templates/mongod.standalone.conf.j2]: The reason for doing this was to remove the necessary bind to the VPN IP that is not present after the reboot.
Remove the if statement `{% if mongodb_internal_vpn_ip is defined %}{{ mongodb_internal_vpn_ip }},{% endif %}` and bind only to the localhost (127.0.0.1)

File: tasks/main.yml [hunk 2]
```diff
@@ -9,12 +9,45 @@
...
+  - src: mongod.repl.conf.j2
+    dest: /opt/mongod.repl.conf
+  - src: mongod.standalone.conf.j2
+    dest: /opt/mongod.standalone.conf
+  tags:
+  - mongodb
+
+- name: copy mongo startup with backup script
+  template:
+    src: start_mongo_with_backup.sh
```
Comment (Ben10k) [inline tasks/main.yml]: The script executes a restart and the name shows that it only starts the mongod service

File: tasks/main.yml [hunk 3]
```diff
@@ -9,12 +9,45 @@
...
+    mode: u=rx,g=r,o=r
+  tags:
+  - mongodb
+
+- name: backup mongod.conf
+  command: cp -a /etc/mongod.conf /etc/mongod.conf.bak
+  tags:
+  - mongodb
+
+- name: remove previous config incase of it being not a symlink
```
Comment (Ben10k) [inline tasks/main.yml]: in case

**Review by Ben10k — CHANGES_REQUESTED** [review]

File: templates/mongod.repl.conf.j2 [hunk 4]
```diff
@@ -8,7 +8,7 @@ processManagement:
   fork: true
 
 net:
-  bindIp: {% if mongodb_internal_vpn_ip is defined %}{{ mongodb_internal_vpn_ip }},{% endif %}127.0.0.1
+  bindIp: 127.0.0.1
```
Comment (Ben10k) [inline templates/mongod.repl.conf.j2]: The if statement must be present in the repl config

**Review by Ben10k — CHANGES_REQUESTED** [review]

**Review by Ben10k — APPROVED** [review]

**Review by d0nce — APPROVED** [review]

**Pull request merged**